test/services/network/translator/test_network_translator.py

```diff
@@ -43,6 +43,13 @@
     # "create_location": create_location(),
     "create_organisation": create_organisation(),
 
+    #####################################
+    # IEC61968 infIEC61968 InfAssetInfo #
+    #####################################
+
+    "create_current_transformer": create_current_transformer(),
```

Comment:
Added to `__all__` in `cim_creators.py`

Reply:
todo: @Lax125 This is telling you that you have overridden the key, rather than the called function is not known (although you did fix that anyway). Should be `"create_current_transformer_info": create_current_transformer_info(),`

Reply:
Replaced with `"create_current_transformer_info": create_current_transformer_info()`. Also defined the function `create_current_transformer_info` (whoops)